test/e2e/common_suite_test.go

```diff
@@ -56,3 +59,147 @@ func doTestCreateSimplePod(t *testing.T, assert CloudAssert) {
 		}).Feature()
 	testEnv.Test(t, simplePodFeature)
 }
+
+func doTestCreatePodWithConfigMap(t *testing.T, assert CloudAssert) {
+	// TODO: generate me.
```

Comment:
Indeed it would be good to have the namespace auto-generated by the e2e framework.

Reply:
Hi @wainersm ,
Updated the code to generate namespace from e2e-framework
Could you please let me know if you need any changes?

Reply:
Cool, thanks for promptly addressing that!
One last thing @sudharshanibm3 , remove the "// TODO: generate me." comment :)
LGTM.

Reply:
Removed "//TODO:generate me" comment 
Thanks!